src/app/_services/script.store.ts

```diff
@@ -6,8 +6,6 @@ interface Scripts {
 export const ScriptStore: Scripts[] = [
   // Zendesk chat widget
   { name: 'zendesk', src: '../../assets/js/zendesk.js' },
-  // Ethnio, current survey: Insights team on asset and image groups
-  { name: 'ethnio-survey', src: '//ethn.io/45815.js' },
```

Comment:
I wasn't really sure if I should leave this here so that its maybe a little easier to add surveys? Or is the `45815` unique to a specific survey and we would have to change it anyways?

Reply:
Usually we just comment it out instead of deleting it.

Reply:
...so that next time we only need to change the survey number.

Reply:
👍 Will fix